asset_manager/user_interface/views.py

```diff
@@ -1,5 +1,30 @@
 from django.shortcuts import render
+from file_manager.models import TagGroup, Tag, Asset
+from django.conf import settings
 
 # Create your views here.
 def index(request):
-     return render(request, 'user_interface/index.html')
+    return render(request, 'user_interface/index.html')
+
+def list(request):
+	tagGroups = TagGroup.objects.all()
+	context = { 'tagGroups': tagGroups }
+	return render(request, 'user_interface/list.html', context)
+
+def tag_group(request, tag_group_id):
+	tagGroup = TagGroup.objects.get(id=tag_group_id)
+	subTags = Tag.objects.filter(group__id=tag_group_id)
+	assets = Asset.objects.filter(tags__in=subTags)
```

Comment:
To fix the duplication issue, adding `.distinct()` to the end of this line seems to resolve the issue.

Reply:
Nice spot @MertonL and looks like a good fix. Are you going to commit the fix? Happy to do it too.

Reply:
Commited!